source/config-file/restrict-outgoing-connections.txt

```diff
@@ -0,0 +1,75 @@
+.. _compass-configure-network-traffic:
+
+==================================
+Block Outgoing Network Connections
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can configure |compass| to not perform outgoing network operations
+other than those to the database. This lets you configure any
+:ref:`edition <compass-feature-table>` of |compass-short| to restrict
+outgoing connections in the same way :ref:`Compass Isolated Edition
+<compass-faq-isolated>` restricts connections.
+
+Consideration
+-------------
+
+When you block outgoing network connections, |compass-short| can't use
```

Comment:
(I know we’re not to look for stylistic issues, so feel free to disregard, but I think our docs don’t typically use contractions)